src/eduid_userdb/actions/db.py

```diff
@@ -51,6 +52,7 @@ def __init__(self, db_uri, db_name='eduid_actions', collection='actions'):
         super(ActionDB, self).__init__(db_uri, db_name, collection)
 
         self._cache = {}
+        self._left_in_cache = {}
```

Comment:
On Thursday, October 22, 2015 07:49:39 AM Enrique P�rez Arnaud wrote:
...
> However, now that we discuss it, a simpler solution occurs to me, which is
> to iterate over the cursor on first read and cache a list with those
> results. I'll do that and see if you like it better.

Yes, that sounds much better. Thanks for explaining =).

/Fredrik